src/lib.rs

```diff
@@ -222,6 +222,31 @@ pub mod web {
         interval_buffered
     };
 
+
+    /// A module containing the IndexedDB API
+    pub mod indexeddb {
+        pub use webapi::indexeddb::{
+            IDBOpenDBRequest,
+            IDBDatabase,
+            IDBRequestSharedMethods,
+            IDBRequest,
+            IDBIndex,
+            IDBObjectStore,
+            IDBTransaction,
+            IDBTransactionMode,
+            IDBFactory,
+            IDBObjectStoreIndexSharedMethods,
+            IDBCursorDirection,
+            IDBRequestReadyState,
+            IDBCursorSharedMethods,
+            IDBCursor,
+            IDBCursorWithValue,
+            IDBAddError
```

Comment:
I'm not sure about this change. The spec uses `IDB` prefixed names, so matching them makes it easier to look up. `Db` makes it less clear these correspond exactly to IndexedDB.

I tried to make this change, but a naive find-and-replace `IDB` to `Db` breaks. First, `#[reference(instance_of = "IDB...")]` needs to be preserved. Second, all the "IDB"s in URLs. It could be made, but would benefit from more powerful refactoring tools, or more tedious careful manual replacements...